build fail looks real:
```
======================================================================
FAIL: testRestoreAttributes (casexml.apps.phone.tests.test_ota_restore.FileRestoreOtaRestoreTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py", line 244, in testRestoreAttributes
    self.assertTrue('<dateattr somedate="2012-01-01">' in restore_payload)
AssertionError: False is not True
----------------------------------------------------------------------
Ran 17 tests in 12.959s
```

--- a/corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py
@@ -7,6 +7,7 @@ from casexml.apps.phone.tests.utils import generate_restore_payload
 from couchforms.tests.testutils import post_xform_to_couch
 from casexml.apps.case.models import CommCareCase
 from casexml.apps.case.tests.util import check_xml_line_by_line, delete_all_cases, delete_all_sync_logs
+from casexml.apps.phone.restore import RestoreConfig
 from casexml.apps.case.xform import process_cases
 from datetime import datetime, date
 from casexml.apps.phone.models import User, SyncLog
@@ -32,6 +33,8 @@ class OtaRestoreTest(TestCase):
     def tearDown(self):
         delete_all_cases()
         delete_all_sync_logs()
+        restore_config = RestoreConfig(dummy_user())
+        restore_config.cache.delete(restore_config._initial_cache_key())
 
     def testFromDjangoUser(self):
         django_user = DjangoUser(username="foo", password="secret", date_joined=datetime(2011, 6, 9))

